client/relay-chain-inprocess-interface/src/lib.rs

```diff
@@ -279,17 +279,23 @@ fn build_polkadot_full_node(
 
 	let relay_chain_full_node = polkadot_service::build_full(
 		config,
-		is_collator,
-		None,
-		// Disable BEEFY. It should not be required by the internal relay chain node.
-		false,
-		None,
-		telemetry_worker_handle,
-		true,
-		polkadot_service::RealOverseerGen,
-		None,
-		None,
-		hwbench,
+		polkadot_service::NewFullParams {
+			is_collator,
+			grandpa_pause: None,
+			// Disable BEEFY. It should not be required by the internal relay chain node.
+			enable_beefy: false,
+			jaeger_agent: None,
+			telemetry_worker_handle,
+			node_version: Some(env!("SUBSTRATE_CLI_IMPL_VERSION").into()),
+			workers_path: None,
+			workers_names: None,
+			dont_use_external_workers: false,
```

Comment:
I think we may need to rig this crate to build the workers for this to work. Can someone please explain what this crate does exactly, does it call PVF preparation/execution?

Reply:
I don't think Cumulus spawns PVF workers. There's some discussion here: https://github.com/paritytech/polkadot/issues/6497